tests/unit/services/variableSetOrderService.spec.ts

```diff
@@ -6,20 +6,19 @@ describe('finalVariableSetSort', () => {
       id: 2,
       subvariables: [ { id: 3 }, { id: 4 }, { id: 5 } ]
     }, {
-      id: 4,
```

Comment:
The spec could really do with a more descriptive name, i can't really figure out what it is doing and if assertions are correct. Maybe add specific cases and describe the case using the case String ( or add description).